README.md

```diff
@@ -32,20 +32,27 @@ CredSweeper is a tool to detect credentials in any directories or files. CredSwe
 
 Stable (release 1.0.0):
 ``` bash
+# Without Ml validation feature
 $ pip install git+https://github.com/Samsung/CredSweeper.git@v1.0.0
+# With Ml validation feature
+$ pip install CredSweeper[ml]@git+https://github.com/Samsung/CredSweeper.git@v1.0.0
```

Comment:
Hm, I think this line should be removed
v1.0.0 setup file do not contain separate Ml extra:

https://github.com/Samsung/CredSweeper/blob/v1.0.0/setup.py

Reply:
Thank you for review.
I removed the line by afb2d45, please check.